### DISCUSSION TITLE
Suggestion: Name change

### DISCUSSION THREAD

**Discussion by dumptruckDS** [andrei-drexler/ironwail#7] [Ideas]
I think it would be great to append your initials to the name of this port or do something similar. Something akin to what Spike has done with Quakespasm-Spiked. Judging by reactions I've already seen, people are going to be very excited about this release when they discover it and it would be nice to differentiate it from other forks. Some lame suggestions:

Quakespasm-Drexler (QSD)
Quakespasm-GL (QSGL)

EDIT: I forgot to mention, I do think this would be important since you kind of have a hybrid version with some features from vkQuake/QSS and the basis being QS. It could get confusing for end users who are looking for the best modern port for their needs.

**Comment by andrei-drexler** [comment]
A new name is definitely on my to-do list, I just haven't been able to come up with anything I really like yet. I'd rather not tack my name onto QS (even if it's common practice), but otherwise I'm definitely open to suggestions, no matter how/especially if silly. Ideally, the name would be
- speed-related
- Quake-related
- medieval-sounding
- unique, easy to search for
- not very long
- silly

but right now coming up with such a name seems as likely as winning the lottery. I should probably stop waiting for inspiration to strike and just pick something.

**Reply by dumptruckDS** [reply]
I missed this response. I'll brainstorm (and ask some folks in Quake Mapping Discord) and post some suggestions here.

**Reply by dumptruckDS** [reply]
Okay I have a small number of really bad ones to inspire you. :)

- Qatapult
- Quakesilver
- ShamblerDance
- BrownLightning
- BrownStreak (LOL maybe not)
- QuakePrix
- Frontrunner
- Qalista
- Qu-Tang
- Qshift
- SupersoniQ
- QuadShift
- JustABlur
- Q-blur
- MotionSiQ
- GLoat
- GLaze
- GibLets

**Reply by andrei-drexler** [reply]
Thanks, really appreciate you taking the time to do this!

**Comment by SecularSteve** [comment]
Let me get in on this:

QTwitch
CodeTetanus
GlassQuake
WarpQuake
MilQuake

**Reply by mjr4077au** [reply]
The project has been known as Ironwail for so long that it'd be remiss to change it now IMHO.

**Reply by DragonKnightKX** [reply]
I also think this might've been a suggestion made before the name Ironwail was thought of.
